crates/driver/src/blockchain/mod.rs

```diff
@@ -12,22 +12,28 @@ pub mod contracts;
 pub struct Ethereum {
     web3: Web3<DynTransport>,
```

Comment:
Should we change the struct name since we support other chains, not only Ethereum? `Blockchain` or similar...

Reply:
For context, there was some discussion about this naming in a previous PR. `Blockchain` was a candidate, but we felt like `Ethereum` better captured the concept of an "EVM-compatible blockchain". For example, Bitcoin is a blockchain, but can't be used here.